### PR TITLE
feat(uat): added new scenario GGAD-1-T8

### PR DESCRIPTION
**Pull request by auarbekov**
**Issue #, if available:**
https://klika-tech.atlassian.net/browse/GGMQ-95
Implement scenario GAAD-1-T8 using existing building blocks

**Description of changes:**
- added  new scenario with 1 ggad.

**Why is this change necessary:**
- required step for several test features

**Test results:**
```
Given my device is registered as a Thing....................................passed
And my device is running Greengrass.........................................passed
And I create client device "publisher"......................................passed
When I associate "publisher" with ggc.......................................passed
When I create a Greengrass deployment with components.......................passed
And I update my Greengrass deployment configuration, setting the component aws.greengrass.clientdevices.mqtt.Bridge configuration to:.passed
And I update my Greengrass deployment configuration, setting the component aws.greengrass.clientdevices.Auth configuration to:.passed
And I update my Greengrass deployment configuration, setting the component aws.greengrass.client.Mqtt5JavaSdkClient configuration to:.passed
And I deploy the Greengrass deployment configuration........................passed
Then the Greengrass deployment is COMPLETED on the device after 300 seconds.passed
When I associate "publisher" with ggc.......................................passed
And I discover core device broker as "default_broker" from "publisher"......passed
And I connect device "publisher" on aws.greengrass.client.Mqtt5JavaSdkClient to "default_broker".passed
When I publish from "publisher" to "topic/to/pubsub" with qos 0 and message "Hello world".passed
When I publish from "publisher" to "topic/device1/humidity" with qos 0 and message "device1 humidity".passed
When I publish from "publisher" to "topic/device2/temperature" with qos 0 and message "device2 temperature".passed
When I publish from "publisher" to "topic/with/prefix" with qos 0 and message "topicPrefix message".passed
```

**Any additional information or context required to review the change:**

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: uat/testing-features/src/main/resources/greengrass/features/ggad-1.feature [hunk 1]
```diff
@@ -0,0 +1,96 @@
+@GGAD
+Feature: GGAD-1
+
+  As a developer, I can configure my GGAD devices to connect to the local GGC using cloud based discovery
```
Comment (bgklika) [inline uat/testing-features/src/main/resources/greengrass/features/ggad-1.feature:4]: Invalid description

File: uat/testing-features/src/main/resources/greengrass/features/ggad-1.feature [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    """
+    And I deploy the Greengrass deployment configuration
+    Then the Greengrass deployment is COMPLETED on the device after 300 seconds
+    When I associate "publisher" with ggc
+    And I discover core device broker as "default_broker" from "publisher"
+    And I connect device "publisher" on aws.greengrass.client.Mqtt5JavaSdkClient to "default_broker"
+    When I publish from "publisher" to "topic/to/pubsub" with qos 0 and message "Hello world"
+    When I publish from "publisher" to "topic/device1/humidity" with qos 0 and message "device1 humidity"
+    When I publish from "publisher" to "topic/device2/temperature" with qos 0 and message "device2 temperature"
+    When I publish from "publisher" to "topic/with/prefix" with qos 0 and message "topicPrefix message"
```
Comment (bgklika) [inline uat/testing-features/src/main/resources/greengrass/features/ggad-1.feature:96]: Missing \n at end of last line.

Comment (auarbekov) [reply uat/testing-features/src/main/resources/greengrass/features/ggad-1.feature]: added empty line in the end.

File: uat/testing-features/src/main/resources/greengrass/features/ggad-1.feature [hunk 3]
```diff
@@ -0,0 +1,96 @@
+@GGAD
+Feature: GGAD-1
```
Comment (esmirnov-klika-tech) [inline uat/testing-features/src/main/resources/greengrass/features/ggad-1.feature:2]: Could you rename GGAD abbreviations, please to GGMQ and merge test scenarios in a single file? ggmq-1.feature

**Review by esmirnov-klika-tech — DISMISSED** [review]

File: uat/testing-features/src/main/resources/greengrass/components/recipes/client_ipc.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+Manifests:
+  - Artifacts:
+      - URI: classpath:/greengrass/artifacts/aws.greengrass.client.IpcClient.jar
+        Permission:
+          Read: ALL
+          Execute: ALL
+    Lifecycle:
+      Run: |
+        java -DcomponentName="LocalIpcSubscriber" -jar {artifacts:path}/aws.greengrass.client.IpcClient.jar {configuration:/topicsToSubscribe}
```
Comment (bgklika) [inline uat/testing-features/src/main/resources/greengrass/components/recipes/client_ipc.yaml]: take arguments into double quotes. 
For example your script will fail when value in {artifacts:path} or {configuration:/topicsToSubscribe} contains space

Must be 
java -DcomponentName="LocalIpcSubscriber" -jar "{artifacts:path}/aws.greengrass.client.IpcClient.jar" "{configuration:/topicsToSubscribe}"

**Review by bgklika — DISMISSED** [review]

File: uat/testing-features/src/main/resources/greengrass/components/recipes/client_ipc.yaml (hunk 4 above)
Comment (esmirnov-klika-tech) [reply uat/testing-features/src/main/resources/greengrass/components/recipes/client_ipc.yaml]: fixed 23fa329e42c9d0841d7aff88c9b977db8d93de1e

File: uat/testing-features/src/main/resources/greengrass/features/ggad-1.feature (hunk 3 above)
Comment (esmirnov-klika-tech) [reply uat/testing-features/src/main/resources/greengrass/features/ggad-1.feature]: fixed

**Pull request closed without merging**